### PR TITLE
feat: add logLevel option

### PR DESCRIPTION
**Pull request by bamthomas**
add a new option to set logback loglevel programmatically.

### CODE REVIEW

File: datashare-app/src/main/java/org/icij/datashare/Main.java [hunk 1]
```diff
@@ -14,7 +15,11 @@ public static void main(String[] args) throws Exception {
         LOGGER.info("Running datashare " + (cli.isWebServer() ? "web server" : ""));
         LOGGER.info("JVM version {}", System.getProperty("java.version"));
         LOGGER.info("JVM charset encoding {}", Charset.defaultCharset());
-        LOGGER.debug("debug logs are activated");
+        Level logLevel = Level.toLevel(cli.properties.getProperty("logLevel"));
+        LOGGER.info("log level set to {}", logLevel);
```
Comment (pirhoo) [inline datashare-app/src/main/java/org/icij/datashare/Main.java:19]:
```suggestion
        LOGGER.info("Log level set to {}", logLevel);
```

File: datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCli.java [hunk 2]
```diff
@@ -9,6 +9,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.xml.crypto.Data;
```
Comment (pirhoo) [inline datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCli.java:12]: Is this needed?

File: datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCliOptions.java [hunk 3]
```diff
@@ -593,4 +594,12 @@ public static OptionSpec<String> extOption(OptionParser parser) {
         return parser.acceptsAll(
                 singletonList("ext"), "Run CLI extension").withRequiredArg().ofType(String.class);
     }
+
+    static void logLevel(OptionParser parser) {
+        parser.acceptsAll(
+                        singletonList("logLevel"),
+                        format("sets the log level of datashare (%s)", Arrays.toString(Level.values())))
```
Comment (pirhoo) [inline datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCliOptions.java:601]:
```suggestion
                        format("Sets the log level of Datashare (%s)", Arrays.toString(Level.values())))
```

**Review by pirhoo — APPROVED** [review]

File: datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCli.java (hunk 2 above)
Comment (bamthomas) [reply datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCli.java:12]: Nope, no idea why it has been added (automatic imports I guess on bad completion)

Comment (bamthomas) [reply datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCli.java]: removed with https://github.com/ICIJ/datashare/pull/1308/commits/52e0305771ce8fb79d5eca2ec5574f221a53f76e

**Pull request merged**